packages/cli/package.json

```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "@emdaer/core": "^1.2.2",
+    "@emdaer/shared": "file:../../shared",
```

Comment:
If someone installs @emdaer/cli will this work, or will this only work when developing emdaer locally?

Reply:
Good point – I'm not sure how best to accomplish this

Reply:
Pulling feature flags into core would fix this